docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -0,0 +1,57 @@
+# Creating a new network from the Cleura Cloud Control Panel
+
+Before creating a server in {{extra.brand}} Cloud, you need to have a 
+network to make the new server a member of. You may have more than one 
+network per region, so let us now walk through creating a new network 
+using the {{extra.brand}} Cloud Control Panel. Fire up your favorite web 
+browser, navigate to the [Cleura Cloud](https://cleura.cloud) page, and 
```

Comment:
This is up for a discussion, but in another docs page, file create-shoot-cluster.md line 13, when including cleura.cloud link, we use `<https://{{config.extra.gui_domain}}>` instead . This is later rendered as a full link on a page.
Both ways are fine, but for consistency reasons, I think it's better to stick with one approach.